index.html

```diff
@@ -1,434 +1,577 @@
+      }
+      
+      /*a.main {*/
+        /*  background-color: #E07A5F;*/
+        /*  color: #3D405B;*/
+        /*  font-family: 'Times New Roman', Times, serif;*/
+        /*  border-radius: 10pt 10pt 10pt 10pt;*/
+        /*  padding: 10pt;*/
+        /*  font-size: 3vw;*/
+        /*  text-decoration: none;*/
+        /*  transition: padding 1s;*/
+        /*  overflow: hidden;*/
+        /*  box-shadow: 2px 2px #3D405B;*/
+        /*}*/
+        
+        
+        /* Setting Navbar a different background image using Pseudoselector named "bg.png" for a better UI and adjusting fonts, logo and color */
+        
+        
+        .navbar::before {
+          content: "";
+          position: absolute;
+          top: 2px;
+          width: 100%;
+          height: 94%;
+          background-image: url("./img/bg.png");
+        opacity: 1;
+        z-index: -1;
+      }
+      
+      .navbar,
+      .container-fluid {
+        background-color: #0a05057a;
+      }
+      
+      .navbar-dark .navbar-brand {
+        color: #bcf3bb;
+        text-shadow: 1px 2px 2px #873bc8;
+        cursor: pointer;
+      }
+      
+      .navbar-dark .navbar-nav .nav-link {
+        color: rgba(255, 255, 255, .95);
+        text-shadow: 2px 2px #37547a;
+        font-size: 1.2rem;
+      }
+
+      .navbar-brand img {
+        position: relative;
+        width: 50px;
+        height: 50px;
+        left: -4px;
+        top: -10px;
+        margin-right: 8px;
+      }
+      
+      
+      /* ***adjustments done */
+      
+      a.hyperlink {
+        color: #e07a5f;
+        font-weight: bold;
+        text-decoration: none;
+      }
+      
+      a.hyperlink:hover {
+        color: orangered;
+        background-color: rgba(76, 0, 130, 0.082);
+      }
+      
+      /* div.gridcontainer {
+        display: grid;
+        grid-column-gap: 1px;
+        grid-template-columns: auto auto auto;
+        padding: 5px;
+        text-align: center;
+      }
+      
+      div.grid-item {
+        text-align: center;
+        padding: 10px;
+        display: block;
+        grid-column-gap: 1px;
+        margin: 5px;
+        line-height: 60px;
+      } */
+      
+      .fade:not(.show) {
+        opacity: 1;
+      }
+
+      .navbar-brand {
+        font-family: "Rubik", sans-serif;
+        font-size: 2.5rem;
+        font-weight: 700;
+        color: white;
+      }
+      
+      .nav-link {
+        font-family: "Rubik", sans-serif;
+        font-size: 1.5rem;
+        font-weight: 200;
+      }
+      
+      .nav-item {
+        padding: 0 18px 0 0;
+      }
+
+      /*a.main:hover {*/
+      /*  text-decoration: underline;*/
+      /*  padding: 15pt;*/
+      /*  transition: padding 1s;*/
+      /*  box-shadow: none;*/
+      /*}*/
+      
+      .slideshow {
+        padding-top: 30px;
+        max-width: 1000px;
+        position: relative;
+        margin: auto;
+      }
+      
+      .SlideShow {
+        display: none;
+      }
```

Comment:
Can you make the names of these CSS classes clearer? Right now they are both named slideshow, only differentiated by capitalization.